front-end/src/components/Dashboard/index.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
```

Comment:
You can ignore this file for now. This is actually part of @PaulNewling's work, but I needed this page to confirm that the login worked